magicctapipe/scripts/lst1_magic/magic_calib_to_dl1.py

```diff
@@ -162,6 +167,38 @@ def magic_calib_to_dl1(input_file, output_dir, config, max_events, process_run=F
             subrun_id = event_source.metadata["subrun_number"][0]
             output_file = f"{output_dir}/dl1_M{tel_id}.Run{obs_id:08}.{subrun_id:03}.h5"
 
+    # Reset the telescope IDs of the subarray description
+    if event_source.is_hast:
+        tel_positions_magic_lst = {
+            1: [-8.09, 77.13, 0.78] * u.m,  # LST-1
```

Comment:
but what happens if somebody uses different tel_ids in config file for HaST data?
maybe we can at least introduce a check that for HaST data lst-1 is 1  etc. ?